test/integration/replicaset/replicaset_test.go

```diff
@@ -27,16 +27,24 @@ import (
 	"k8s.io/api/core/v1"
 	"k8s.io/api/extensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/informers"
 	clientset "k8s.io/client-go/kubernetes"
 	restclient "k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/retry"
+	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
 	"k8s.io/kubernetes/pkg/controller/replicaset"
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
+const (
+	interval = 100 * time.Millisecond
+	timeout  = 5 * time.Second
```

Comment:
@enisoc I am not sure what is the best timeout duration for the tests. One second is too short (some polls will time out). Currently, it takes around 35 seconds to run all RS integration tests (including those that are not introduced by this PR).

Reply:
Go with the previous value of 60*time.seconds. The integration tests have been using this value so far and have not been flakey. We expect the test to pass, so we should not often timeout and have to wait for the full 60 seconds.

Reply:
noted, i will change the timeout to `60 * time.Second`, but still have the interval intact.